src/fsharp/ast.fs

```diff
@@ -6,12 +6,12 @@ open System.Collections.Generic
 open Internal.Utilities
 open Internal.Utilities.Text.Lexing
 open Internal.Utilities.Text.Parsing
-open Microsoft.FSharp.Compiler.AbstractIL 
-open Microsoft.FSharp.Compiler.AbstractIL.IL 
-open Microsoft.FSharp.Compiler.AbstractIL.Internal 
-open Microsoft.FSharp.Compiler.AbstractIL.Internal.Library 
-open Microsoft.FSharp.Compiler 
-open Microsoft.FSharp.Compiler.UnicodeLexing 
+open Microsoft.FSharp.Compiler.AbstractIL
+open Microsoft.FSharp.Compiler.AbstractIL.IL
+open Microsoft.FSharp.Compiler.AbstractIL.Internal
+open Microsoft.FSharp.Compiler.AbstractIL.Internal.Library
+open Microsoft.FSharp.Compiler
+open Microsoft.FSharp.Compiler.UnicodeLexing
```

Comment:
Probably best to minimize the diff be removing all unnecessary whitespace-only changes (there are quite a few of those)